app/views/users/registrations/i_downloaded.erb

```diff
@@ -33,7 +33,7 @@
           <span class="glyphicon glyphicon-download mr-2"></span>
           Export Report
         </button>
-        <%= link_to 'Back', users_path, :class => "btn button_user_card ml-2"  %>           
+        <%= link_to 'Back', search_page_path, :class => "btn button_user_card ml-2"  %>           
```

Comment:
The 'I downloaded' usually comes from the show page so redirect to `user_path(@user.id)`.  (Unless you want to get fancy and check to see if the call came from the users index with the use of parameters)